Src/PerformanceCollector/Perf.Shared/Implementation/WebAppPerformanceCollector/CacheHelper.cs

```diff
@@ -1,7 +1,12 @@
 ﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.WebAppPerformanceCollector
 {
     using System;
+    using System.Collections.Generic;
+#if NETSTANDARD1_6
+    using Microsoft.Extensions.Caching.Memory;
+#else
```

Comment:
Synced with sergey about this. We want to minimize adding additional dependencies for web sdk in .net framework target. So keeping existing behavior.
For netstandard, we take a new dependency of the new CacheProvider, and hence code is IF_DEF ed.